core/shared/src/main/scala/com/lynbrookrobotics/potassium/Component.scala

```diff
@@ -54,6 +58,10 @@ abstract class Component[T](period: Time)(implicit val clock: Clock) {
     }
     // scalastyle:on
 
+    val value = currentController.currentValue(dt)
+
+    lastOutput = Some(value)
```

Comment:
is "value" public by intention? I'd just do 
Some(currentController.currentValue(dt))